apps/web/hooks/useDashboardChart.ts

```diff
@@ -31,9 +36,14 @@ function useDashboardChart() {
     fetcher,
   );
 
+  // --- Derived data ---
   const { chartOptions, isEmpty } = useMemo(() => {
     const categories = result?.data?.map((item) => item.sentiment) ?? [];
-    const data = result?.data?.map((item) => item.count) ?? [];
+    const data =
```

Comment:
create a helper function to transform the result from response to format of data expected by chart library

Reply:
And assign proper types